platforms/documentation/docs/src/docs/userguide/releases/upgrading/upgrading_version_8.adoc

```diff
@@ -140,28 +140,31 @@ Please refer to the <<configuration_cache#config_cache:requirements:use_project_
 [[groovy_space_assignment_syntax]]
 === Groovy "space assignment" syntax
 
-Currently, there are multiple ways to set a property with Groovy DSL syntax:
+Currently, in the Groovy DSL, you can set a property using several syntaxes:
 
-[source]
----
+[source,groovy]
+----
 propertyName = value
 setPropertyName(value)
 setPropertyName value
 propertyName(value)
 propertyName value
----
+----
+
+The last option, _space-assignment_, is specific to Gradle and not part of standard Groovy.
+
+In standard Groovy, `propertyName(value)` is just a method call.
+Gradle dynamically generates the `propertyName` method at runtime if it doesn't already exist.
 
-The latter one, "space-assignment", is a Gradle-specific feature that is not part of the Groovy language.
-In regular Groovy, this is just a method call: `propertyName(value)`, and Gradle generates `propertyName` method in the runtime if this method hasn't been present already.
 This feature may be a source of confusion (especially for new users) and adds an extra layer of complexity for users and the Gradle codebase without providing any significant value.
 Sometimes, classes declare methods with the same name, and these may even have semantics that are different from a plain assignment.
 
-These generated methods are now deprecated and will be removed in Gradle 10.0, and both `propertyName value` and `propertyName(value)` will stop working unless the explicit method `propertyName` is defined.
+These generated methods are now deprecated and will be removed in Gradle 10.0.
+Both `propertyName value` and `propertyName(value)` will stop working unless the explicit method `propertyName` is defined.
 Use explicit assignment `propertyName = value` instead.
 While explicit methods can be used with space, like `propertyName value`, consider using the `propertyName(value)` syntax for clarity.
-For example, `jvmArgs "some", "arg"` can be replaced with `jvmArgs("some", "arg")` or with `jvmArgs = ["some", "arg"]` for `Test` tasks.
 
-[[deprecated_configuration_cache_options]]
```

Comment:
@ov7a should there be something in this section or was this added by mistake?

Reply:
It was added by accident. I'm not sure what happened, maybe it's an artifact of rebase. This should be removed, but I actually did that already in [this](https://github.com/gradle/gradle/pull/31517/files) PR, that I sent for merge